src/main/kotlin/org/wfanet/measurement/measurementconsumer/stats/MeasurementStatistics.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2023 The Cross-Media Measurement Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wfanet.measurement.measurementconsumer.stats
+
+import org.wfanet.measurement.eventdataprovider.noiser.DpParams
+
+/** Interface that contains necessary information about computing the measurement. */
+sealed interface MeasurementParams
```

Comment:
I know nothing about `sealed interface`. you probably dont need my review of this file, but please point it to me if you'd like to.